CriticalMass/ChatNavigationButton.swift

```diff
@@ -24,9 +32,7 @@ class ChatNavigationButton: CustomButton {
     init() {
         super.init(frame: .zero)
         setImage(UIImage(named: "Chat")!, for: .normal)
-        tintColor = .navigationOverlayForeground
         adjustsImageWhenHighlighted = false
-        highlightedTintColor = UIColor.navigationOverlayForeground.withAlphaComponent(0.4)
```

Comment:
The button doesn't update its color anymore for the pressed state

Reply:
fixed that. check out latest changes